### PR TITLE
fix(DB/Zones):improved missing & skipped some NPCs

### PR DESCRIPTION
**Pull request by Knindzagxg**
Improvements to few missing and skipped NPCs from before.
Also, updated few spawns.

To test it:
Apply SQL
Restart your server

You can manually spawn NPC by ID to check SmartAI or you can go to zone and check like that as well.

.npc add ID

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1601315140580867400.sql [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+(4002,0,0,0,2,0,100,1,0,15,0,0,25,1,0,0,0,0,0,1,0,0,0,0,0,0,0,'Windshear Stonecutter - Between 0-15% Health - Flee For Assist (No Repeat)');
+
+UPDATE `creature_template` SET `AIname`='SmartAI' WHERE `entry`=11912;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=11912 AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(11912,0,0,0,4,0,100,0,0,0,0,0,11,6268,0,0,0,0,0,2,0,0,0,0,0,0,0,'Grimtotem Brute - On Aggro - Cast RushIng Charge'),
+(11912,0,1,0,2,0,100,1,0,15,0,0,25,1,0,0,0,0,0,1,0,0,0,0,0,0,0,'Grimtotem Brute - Between 0-15% Health - Flee For Assist (No Repeat)');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=17236; CHECk
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1601315140580867400.sql:89]: mistake

Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1601315140580867400.sql:89]: Its was about last part to add despawn on evade but forgot to remove check xP

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by pklloveyou** [comment]
Tested

File: data/sql/updates/pending_db_world/rev_1601315140580867400.sql [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+(4011,0,0,0,0,0,100,0,5000,8000,12000,18000,11,744,32,0,0,0,0,2,0,0,0,0,0,0,0,'Young Pridewing - In Combat - Cast Poison');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=4263;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=4263 AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(4263,0,0,0,2,0,50,1,0,30,0,0,11,6536,0,0,0,0,0,1,0,0,0,0,0,0,0,'Deepmoss Hatchling - At 30% HP - Summon Deepmoss Matriarch');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=2172;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=2172;
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1601315140580867400.sql:20]: this is dangerous! missing `source_type` can lead to deleting unwanted contents!

File: data/sql/updates/pending_db_world/rev_1601315140580867400.sql [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2172,0,0,0,0,0,100,0,7000,11000,12000,14000,11,7272,0,0,0,0,0,1,0,0,0,0,0,0,0,'Strider Clutchmother - In Combat - Cast Dust Cloud'),
+(2172,0,1,0,9,0,100,0,0,5,15000,17000,11,6607,0,0,0,0,0,2,0,0,0,0,0,0,0,'Strider Clutchmother - Within 0-5 Range - Cast Lash'),
+(2172,0,2,0,2,0,100,1,0,15,0,0,25,1,0,0,0,0,0,1,0,0,0,0,0,0,0,'Strider Clutchmother - Between 0-15% Health - Flee For Assist (No Repeat)');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=2170;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=2170;
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1601315140580867400.sql:27]: this is dangerous! missing `source_type` can lead to deleting unwanted contents!

File: data/sql/updates/pending_db_world/rev_1601315140580867400.sql [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2170,0,0,0,14,0,100,0,200,40,18000,21000,11,1058,0,0,0,0,0,7,0,0,0,0,0,0,0,'Blackwood Ursa - Friendly At 200 Health - Cast Rejuvenation');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=2171;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=2171;
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1601315140580867400.sql:32]: this is dangerous! missing `source_type` can lead to deleting unwanted contents!

File: data/sql/updates/pending_db_world/rev_1601315140580867400.sql [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2171,0,0,0,0,0,100,0,1000,2000,3000,3400,11,9532,64,0,0,0,0,2,0,0,0,0,0,0,0,'Blackwood Shaman - In Combat CMC - Cast Lightning Bolt'),
+(2171,0,1,0,0,0,100,0,9000,14000,11000,15000,11,2606,0,0,0,0,0,2,0,0,0,0,0,0,0,'Blackwood Shaman - In Combat - Cast Shock'),
+(2171,0,2,0,14,0,100,0,300,40,16000,21000,11,913,0,0,0,0,0,7,0,0,0,0,0,0,0,'Blackwood Shaman - Friendly At 300 Health - Cast Healing Wave');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=2186;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=2186;
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1601315140580867400.sql:39]: this is dangerous! missing `source_type` can lead to deleting unwanted contents!

File: data/sql/updates/pending_db_world/rev_1601315140580867400.sql [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2186,0,0,0,25,0,100,0,0,0,0,0,75,8876,0,0,0,0,0,1,0,0,0,0,0,0,0,'Carnivous the Breaker - On Respawn - Cast Thrash (No Repeat)'),
+(2186,0,1,0,9,0,100,0,0,0,15000,20000,11,6016,0,0,0,0,0,2,0,0,0,0,0,0,0,'Carnivous the Breaker - Within 0-5 Range - Cast Pierce Armor');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=2183;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=2183;
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1601315140580867400.sql:45]: this is dangerous! missing `source_type` can lead to deleting unwanted contents!

File: data/sql/updates/pending_db_world/rev_1601315140580867400.sql [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2183,0,0,0,0,0,100,0,4000,8000,10000,12000,11,3248,0,0,0,0,0,1,0,0,0,0,0,0,0,'Stormscale Warrior - In Combat - Cast Improved Blocking');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=7016;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=7016;
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1601315140580867400.sql:50]: this is dangerous! missing `source_type` can lead to deleting unwanted contents!

File: data/sql/updates/pending_db_world/rev_1601315140580867400.sql [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(7016,0,0,0,0,0,100,0,1500,3000,3000,3100,11,6660,64,0,0,0,0,2,0,0,0,0,0,0,0,'Lady Vespira - In Combat - Cast Shoot'),
+(7016,0,1,0,9,0,100,0,0,5,9000,14000,11,11428,1,0,0,0,0,2,0,0,0,0,0,0,0,'Lady Vespira - Within 0-5 Range - Cast Knockdown'),
+(7016,0,2,0,0,0,100,0,11000,15000,14000,17000,11,12549,1,0,0,0,0,2,0,0,0,0,0,0,0,'Lady Vespira - In Combat - Cast Forked Lightning');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=2175;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=2175;
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1601315140580867400.sql:57]: this is dangerous! missing `source_type` can lead to deleting unwanted contents!

**Comment by pklloveyou** [comment]
I checked that the original content was empty

**Comment by Knindzagxg** [comment]
Sorted Shin.

Weird that happened tho

**Comment by Si1ker** [comment]
@Knindzagxg I'll properly test tonight. 🐱

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
waiting for @Si1ker feedback on this

**Comment by Knindzagxg** [comment]
Cheers mate.

Dunno why, but first time Keira2 failed xP
Time to switch to new version xP

**Comment by FrancescoBorzi** [comment]
> Dunno why, but first time Keira2 failed xP
> Time to switch to new version xP

indeed :D

**Review by FrancescoBorzi — APPROVED** [review]

**Review by iThorgrim-Hub — APPROVED** [review]

**Pull request merged**